docs/src/FAQ.md

```diff
@@ -76,3 +76,105 @@ ChainRulesTestUtils.jl has some dependencies, so it is a separate package from C
 This means your package can depend on the light-weight ChainRulesCore.jl, and make ChainRulesTestUtils.jl a test-only dependency.
 
 Remember to read the section on [On writing good `rrule` / `frule` methods](@ref).
+
+## How do chain rules work for complex functions?
+
+`ChainRules.jl` follows the convention that `frule` applied to a function ``f(x + i y) = u(x,y) + i v(x,y)`` with perturbation ``\Delta x + i \Delta y`` returns the value and
```

Comment:
Super pedantic, but elsewhere, "ChainRules.jl" is not in fixed-width font.
```suggestion
ChainRules.jl follows the convention that `frule` applied to a function ``f(x + i y) = u(x,y) + i v(x,y)`` with perturbation ``\Delta x + i \Delta y`` returns the value and
```

Reply:
:+1: 
Also removed the .jl.